lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -41,7 +41,11 @@ public function handleError($code, $message, $file = '', $line = 0, $context = a
 
     public function install()
     {
-        $this->existingHandler = set_error_handler(array($this, 'handleError'), E_ALL);
+        $this->existingHandler = set_error_handler(
```

Comment:
Yes. More than that.
You can read files with
```php
$fh = fopen('/tmp/test.zip', 'r');
$data = '';
while(!feof($fh)) {
    $data .= fread($fh, 1024);
}
fclose($fh);
```

But I think you just use file_get_contents. And I think we want to use **modern** language. So why should we use some things which was introduced because of some guys who doesn\'t know how to  develop programming language?